postgres/src/main/kotlin/com/example/plugins/Routing.kt

```diff
@@ -0,0 +1,81 @@
+package com.example.plugins
+
+import com.example.models.Article
+import com.example.service.ArticleService
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.request.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import kotlinx.coroutines.*
+import java.sql.*
+
+fun Application.configureRouting() {
+    val dbConnection: Connection = connectToPostgres(embedded = false)
+    val articleService = ArticleService(dbConnection)
+    routing {
+        // Create new Article
+        post("/articles") {
+            val article = call.receive<Article>()
+            val id = articleService.create(article)
+            call.respond(HttpStatusCode.Created, id)
+        }
+        // Read an Article
+        get("/articles/{id}") {
+            val id = call.parameters["id"]?.toInt() ?: throw IllegalArgumentException("Invalid ID")
+            try {
+                val article = articleService.read(id)
+                call.respond(HttpStatusCode.OK, article)
+            } catch (e: Exception) {
```

Comment:
`cause: Throwable`

Reply:
Here we should probably catch only specific exceptions and only for the database call.